### DISCUSSION TITLE
New game mode: Chess

### DISCUSSION THREAD

**Discussion by InterestingBrainPoops** [BattlesnakeOfficial/feedback#78] [General]
TLDR: The time controls would be adjusted to match those of chess. Basically you start off with a certain amount of time, and it subtracts how long it took you to move each turn. Running out of time would lead to you timing out for the rest of the game.

Movement: Would not be changed
Board Layout: Would not be changed
Food: Would not be changed
Health: Would not be changed

Why:
As battlesnakes get more complex, they need a lot more time. A key feature of battlesnake is trying to figure out what you can do in the time given, and that is still present in this system. I personally think that strategy could change a lot if snakes were given more time.
